aws/flatcar-linux/kubernetes/require.tf

```diff
@@ -2,24 +2,12 @@
 
 terraform {
   required_version = ">= 0.12.0"
-}
-
-provider "aws" {
-  version = "2.31.0"
-}
-
-provider "local" {
-  version = "~> 1.2"
-}
-
-provider "null" {
-  version = "~> 2.1"
-}
-
-provider "template" {
-  version = "~> 2.1"
-}
 
-provider "tls" {
-  version = "~> 2.0"
+  required_providers {
+    aws      = "2.31.0"
+    local    = "~> 1.2"
+    null     = "~> 2.1"
+    template = "~> 2.1"
+    tls      = "~> 2.0"
```

Comment:
no ct provider needs to be added here? (I see you added it on bare-metal)

Reply:
It wasn't there, but yeah, it should be added. Adding in a separate commit.